detection_rules/exception.py

```diff
@@ -0,0 +1,143 @@
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License
+# 2.0; you may not use this file except in compliance with the Elastic License
+# 2.0.
+"""Rule exceptions data."""
```

Comment:
Will you update the current `exceptions_list: Optional[list]` in rule.py with these classes?